app/assets/javascripts/accordion-with-descriptions.js

```diff
@@ -18,7 +17,7 @@
       var $openOrCloseAllButton;
       var GOVUKServiceManualTopic = serviceManualTopicPrefix();
 
-      //addOpenCloseAllButton();
```

Comment:
If this is not being used can we delete it rather than commenting it out?

Reply:
Sure thing - I'll bundle this into another 'neatening code' type pull request separate to this one.